docker-compose.yaml

```diff
@@ -0,0 +1,28 @@
+services:
+  db:
+    image: postgres:14.15-alpine3.21
+    restart: always
+    env_file:
+      - .env
+    volumes:
+      - postgres_data:/var/lib/postgresql/data
+
+  app:
+    build: .
+    command: >
+      sh -c "python manage.py wait_for_db &&
+            python manage.py migrate &&
+            python manage.py runserver 0.0.0.0:8000"
+    volumes:
+      - ./:/app
+      - media_volume:/app/media
+    ports:
+      - "8000:8000"
+    env_file:
+      - .env
+    depends_on:
+      - db
```

Comment:
The `depends_on` option ensures that the `db` service is started before the `app` service. However, it does not wait for the database to be ready. Ensure that your application handles database readiness, which seems to be addressed by the `wait_for_db` command.